pkg/consts/constants.go

```diff
@@ -112,6 +112,8 @@ const (
 	KernelArgIommuPt    = "iommu=pt"
 
 	ParallelNicConfigFeatureGate = "parallelNicConfig"
+
+	InfinibandGUIDConfigFilePath = "/host/var/opt/infiniband_guids"
```

Comment:
Let's drop `/host` from the path. You need to use https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/d9bbbb6d4ccba4e3a750a6c36d05af239fcd4552/pkg/utils/utils.go#L117 one of this function when you use the value of the constatnt